src/views/nft/components/CollectibleCard/PlaceholderImage.tsx

```diff
@@ -0,0 +1,10 @@
T8hTNWP8HYDonRr4ac+BMSpzTkLUyNzFaVGBmkbkwXEaTJZ8W8QpuJHdHiBonTYyCvaFhCntuYrTJXPU5QqH6DlJxMQp2SU2U8kTNlHMH8BojTfzjvbFBCnNuYqTJXOUZQqHZxlry4gTqsTr34BYVqdOP0FRCm9qTO2JSBOdc9TmCqbnyhVNjDLzSYgTtnoF19YmBYTbncCUdrO2pXaEBCnOucoTJXMTZQqGZRlFicgTsWN5OCChOkgUf4XiFL+GVhB3QLiVNf8hKnweYlS4QOyvGoExKmaUQ3CVPCsRKng4VhalQLiVMfYhKnQOYlSoYOxrOoFxKn8EQpTgTO6idLx25fDp49nw7ez7wWu0JII1C0gTmXPT5gKm0+I0ufrKJ2LUmHTsZyWBMSp3GkKU0GzEaWChmEpXQiIU5ljFqZC5iJKhQzCMroTEKfyRi5MBcxElAoYgiV0LSBOZY1fmDLPQ5QyD8DlCfwjIE7l3ArClHEWopQR36UJ7BAQpx0oGR4SpgzoN5cUpUzwLkvggIA4HQDa4Glh2gD54SVE6aGI3wmUJSBOeechTBv7i9LG4C5HYKaAOM2ES/A2YUqAOPYUojRWyusIlCEgTnnmIEwbuYvSRtAuQyCxQIjTzV8P9uXTeeKzO90uAWHapZL4MVFKDOp0BDYWuInT+9/eDGen34bTL982vnp/lxOmlWcuSisDOz2BjQSeP382vPv1tTht4C1MKyKL0oq4Tk0gg4A4bYMuTCs5i9JKsE5LILOAOK0/AGFawViUVkB1SgIFCYjTusMQpsS+opQY1OkIFCogTusNRpgS2opSQkynIlCBgDitMyRhSuQqSokgnYZAZQLilH5gwpTAVJQSIDoFgYoFxCnt8IRpoacoLQT0dgKNCIhTukEK0wJLUVqA560EGhQQpzRDFaaZjqI0E87bCDQuIE7LByxMMwxFaQaatxDoSECclg1bmCb6idJEMC8n0KmAOM0fvDBNsBOlCVheSoDAIE7zbgJhGukmSiOhvIwAgXsC4nSPY9QvwjSCSZRGIHkJAQJ7BcRpL83OJ4RpJ8vtg6J0a+EnAgTmC4jTeDthesJKlJ7A8RQBApMFxGkcmTDtcRKlPTAeJkBgkYA4HeYTph1GorQDxUMECCQTEKenKYXpgY8oPQDxKwECqwiI035WYbpjI0p3MPxIgMDqAuK0m1iY/nERpd03iEcJEFhXQJwe+wrTtYkoPb4xPEKAwHYC4nTfuvswidL9G8JvBAjkERCnW/euwyRKtzeCnwgQyC8gTn/PoNswiVL+D6EVECDwWECchqHLMInS4w+DRwgQKEeg9zh1F6ZXxz8Nb395NXz+eDacn30v5060EgIECNwR6DlOXYXp5eufhp/fidKde9+PBAgULNBrnLoJU4jSl0/nw9mf3wq+FS2NAAECtwI9xqmLMIUoff38bTg9Pb+duJ8IECBQgUBvcWo+TKJUwafOEgkQOCjQU5yaDpMoHbzXvYAAgYoEeolTs2ESpYo+bZZKgMBogR7i1GSYRGn0Pe6FBAhUKNB6nJoLkyhV+CmzZAIEJgu0HKemwiRKk+9tbyBAoGKBVuPUTJhEqeJPl6UTIDBboMU4NREmUZp9T3sjAQINCLQWp+rDJEoNfKpsgQCBxQItxanqMInS4nvZCQgQaEiglThVGyZRaujTZCsECCQTuInT+9+Oh69fzofTL3X+vaBVhkmUkt3DTkSAQIMC8Tuy0jhVF6YI7i9kbfDjZEsECKQSiN+VFcapqjBFaFFKde86DwECDQvE78zK4lRNmCKwKDX8MbI1AgRSC8TvzoriVEWYIqwopb5nnY8AgQ4E4ndoJXEqPkwRVJQ6+PjYIgECawnE79IK4lR0mCKkKK11rzovAQIdCcTv1MLjVGyYIqAodfSxsVUCBNYWiN+tBcepyDBFOFFa+x51fgIEOhSI37GFxqm4MEUwUerw42LLBAhsJRC/awuMU1FhilCitNW96ToECHQsEL9zC4tTMWGKQKLU8cfE1gkQ2FogfvcWFKciwhRhRGnre9L1CBAgMMTv4ELilD1MEUSUfDwIECCQTSB+FxcQp6xhihCilO1mdGECBAgEgfidnDlO2cIUAUQp3BOOBAgQyC4Qv5szxilLmOLGRSn7TWgBBAgQeCgQv6MzxWnzMMUNi9LDe8HvBAgQKEYgfldniNOmYYobFaVibj4LIUCAwD6B+J29cZw2C1PcoCjtuwc8ToAAgeIE4nf3hnHaJExxY6JU3E1nQQQIEDgkEL/DN4rT6mGKGxKlQ7P3PAECBIoViN/lG8Rp1TDFjYhSsTebhREgQGCsQPxOXzlOq4UpbkCUxs7c6wgQIFC8QPxuXzFOq4QpLlyUir/JLJAAAQJTBeJ3/EpxSh6muGBRmjprrydAgEA1AvG7foU4JQ1TXKgoVXNzWSgBAgTmCsTv/MRxShamuEBRmjtj7yNAgEB1AvG7P2GckoTp+fNnw/vfjoevolTdTWXBBAgQWCqQOk6Lw3QTpXe/vh7OvnwfTk/Pl+7P+wkQIECgQoGUcVoUJlGq8O6xZAIECKwkkCpOs8MkSitN1mkJECBQsUCKOM0KkyhVfNdYOgECBFYWWBqnyWESpZUn6vQECBBoQGBJnCaFSZQauFtsgQABAhsJzI3T6DCJ0kaTdBkCBAg0JDAnTqPCJEoN3SW2QoAAgY0FpsbpYJhEaeMJuhwBAgQaFJgSpyfDJEoN3h22RIAAgUwCY+O0N0yilGlyLkuAAIGGBcbEaWeYRKnhu8LWCBAgkFkgxOnzydlw/vX7o9U8CpMoPTLyAAECBAgkFngqTvfCJEqJ5Z2OAAECBPYKvDp+Mbz95eXw8E9OMUyitNfOEwQIECCwksDxm5fD8c8v7sXprzCJ0kriTkuAAAECBwUexunof7+fXvn3lA66eQEBAgQIrChwN05Hl5eXV/6RvxW1nZoAAQIERgmEOP0fz9XbAXkyPOEAAAAASUVORK5CYII="
```

Comment:
Likely this is temporary.

Reply:
WTF !!!

Reply:
Hopefully we can get an SVG but this technique is good for ensuring the placeholder image is immediately available.